plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/sootbridge/SootRunner.java

```diff
@@ -1,312 +1,310 @@
-		final IWorkspace workspace = ResourcesPlugin.getWorkspace();
-		try {
-			final List<String> urls = new ArrayList<>();
-			final URI uriString = workspace.getRoot().getFile(javaProject.getOutputLocation()).getLocationURI();
-			urls.add(new File(uriString).getAbsolutePath());
-			return urls;
-		}
-		catch (final Exception e) {
-			Activator.getDefault().logError(e, "Error building project classpath");
-			return Lists.newArrayList();
-		}
-	}
-
-	private static Collection<String> libraryClassPath(IJavaProject project, Boolean dependencyAnalyser) {
-		IPreferenceStore store = Activator.getDefault().getPreferenceStore();
-
-		Collection<String> libraryClassPath = Sets.newHashSet();
-		IClasspathEntry[] rentries;
-		try {
-			// check if "include dependencies" checkbox is checked in preference page or analysis is running for dependencies
-			if (store.getBoolean(Constants.ANALYSE_DEPENDENCIES) || dependencyAnalyser) {
-
-				rentries = project.getRawClasspath();
-				for (IClasspathEntry entry : rentries) {
-					resolveClassPathEntry(entry, libraryClassPath, project);
-				}
-			}
-		}
-		catch (CoreException e1) {
-			e1.printStackTrace();
-		}
-		return libraryClassPath;
-	}
-
-	private static void resolveClassPathEntry(IClasspathEntry entry, Collection<String> libraryClassPath, IJavaProject project) {
-		IClasspathEntry[] rentries;
-		switch (entry.getEntryKind()) {
-			case IClasspathEntry.CPE_SOURCE:
-				libraryClassPath.addAll(applicationClassPath(project));
-				break;
-			case IClasspathEntry.CPE_PROJECT:
-				IJavaProject requiredProject = JavaCore.create((IProject) ResourcesPlugin.getWorkspace().getRoot().findMember(entry.getPath()));
-				try {
-					rentries = project.getRawClasspath();
-					for (IClasspathEntry e : rentries) {
-						resolveClassPathEntry(e, libraryClassPath, requiredProject);
-					}
-
-				}
-				catch (JavaModelException e1) {
-					Activator.getDefault().logError(e1);
-				}
-				break;
-			case IClasspathEntry.CPE_LIBRARY:
-
-				if (entry.getPath().segment(0).equals(project.getProject().getName())) {
-					libraryClassPath.add(project.getProject().getParent().getRawLocation() + Constants.innerFileSeparator + entry.getPath().toOSString());
-				} else {
-					libraryClassPath.add(entry.getPath().toOSString());
-				}
-				break;
-			case IClasspathEntry.CPE_VARIABLE:
-				// JRE entry
-				break;
-			case IClasspathEntry.CPE_CONTAINER:
-				try {
-					IClasspathContainer container = JavaCore.getClasspathContainer(entry.getPath(), project);
-					IClasspathEntry[] subEntries = container.getClasspathEntries();
-					for (IClasspathEntry subEntry : subEntries) {
-						resolveClassPathEntry(subEntry, libraryClassPath, project);
-					}
-				}
-				catch (JavaModelException e) {
-					Activator.getDefault().logError(e);
-				}
-				break;
-		}
-	}
-
-}
+/********************************************************************************
```

Comment:
Same as above. I've not reviewed this file because of how Github shows the diff. Could you please briefly elaborate what you have modified here? Thanks.

Reply:
In this class I replaced the CryptoAnalysis rule transformtion by the own plugin transformtion of the class CrySLModelReader.